packages/gatsby-cli/src/create-cli.js

```diff
@@ -92,8 +94,8 @@ function buildLocalCommands(cli, isLocalSite) {
         .option(`p`, {
           alias: `port`,
           type: `string`,
-          default: `8000`,
-          describe: `Set port. Defaults to 8000`,
+          default: defaultDevelopPort,
+          describe: `Set port. Defaults to ${defaultDevelopPort}`,
```

Comment:
Instead of outputting the value of `defaultDevelopPort`, maybe this could be the same as the README? e.g.  "the PORT environment variable or 8000"

Reply:
What about something like this, which shows the current option that would be used along with the information about how it was chosen?

Host:
    
    describe: `Set host. Defaults to ${defaultHost} (the HOST environment variable or localhost if unset).`,
Port:
    
    describe: `Set port. Defaults to ${defaultDevelopPort} (the PORT environment variable or 8000 if unset).`,